config/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ install(FILES
     lxqt.conf
     session.conf
     windowmanagers.conf
+    waylandwindowmanagers.conf
```

Comment:
Install it to  `${CMAKE_INSTALL_DATADIR}/lxqt/wayland` would be more logical. Would it create issues  to load it?

Reply:
It wouldn't create issues if I change the code, but why?

Reply:
If you mean it because `lxqt-wayland-session` puts its files into that folder, that's more reason for limiting it to `lxqt-wayland-session` and not putting anything else in it, especially because it contains compositor-specific files.

I think the most natural place is the current folder. X11 and Wayland and treated on the same level.

Reply:
Because all other wayland configs are there. But actually the folder is created by `lxqt-wayland-session` afaik.

Reply:
> But actually the folder is created by lxqt-wayland-session afaik.

And my previous comment holds ;)